libraries/wallet/include/scorum/wallet/wallet.hpp

```diff
@@ -345,8 +347,8 @@ class wallet_api
      *  @param json_meta JSON Metadata associated with the new account
      *  @param broadcast true if you wish to broadcast the transaction
      */
-    annotated_signed_transaction create_account(
-        string creator, string new_account_name, string json_meta, bool broadcast);
+    annotated_signed_transaction
```

Comment:
General rule to this file: objects should be passed by const reference. Needs in refactoring